packages/google-java-format/package.yaml

```diff
@@ -12,7 +12,49 @@ categories:
 source:
   id: pkg:github/google/google-java-format@v1.28.0
   asset:
-    file: google-java-format-{{ version | strip_prefix "v" }}-all-deps.jar
+    - target: darwin_arm64
+      file: google-java-format_darwin-arm64
+      bin: google-java-format_darwin-arm64
+    - target: linux_arm64
+      file: google-java-format_linux-arm64
+      bin: google-java-format_linux-arm64
+    - target: linux_x64
+      file: google-java-format_linux-x86-64
+      bin: google-java-format_linux-x86-64
+    - target: win_x64
+      file: google-java-format_windows-x86-64.exe
+      bin: google-java-format_windows-x86-64.exe
+    # All other platforms that don't have binary
+    - target: [
+        unix,
+        darwin,
+        darwin_x64,
+        linux,
+        linux_arm,
```

Comment:
Not totally sure about it, but I think you don't need to list every platform here, I think the generic ones like `linux` suffice. 

(You can test it by keeping it your own platform and see what artifact gets downloaded for you)